client/trino-client/src/test/java/io/trino/client/auth/external/TestExternalAuthenticator.java

```diff
@@ -352,8 +354,14 @@ ThrowableAssert firstException()
 
         void assertThatNoExceptionsHasBeenThrown()
         {
-            assertThat(exceptions)
-                    .isEmpty();
+            try {
```

Comment:
Please revert

Reply:
It's important I feel, as without this we know nothing about the exceptions that has occurred during the test execution, so it's quite important to have this now, just to be prepared for the next round of flaky test, especially that anything that will come up right now, won't be easy to reproduce locally.